client/indexer-db/src/models/bsp.rs

```diff
@@ -0,0 +1,92 @@
+use bigdecimal::BigDecimal;
+use chrono::NaiveDateTime;
+use diesel::prelude::*;
+use diesel_async::RunQueryDsl;
+
+use crate::{
+    models::multiaddress::MultiAddress, schema::bsp, schema::bsp_multiaddress, DbConnection,
+};
+
+/// Table that holds the BSPs.
+/// The account is guaranteed to be unique across both MSPs and BSPs.
+#[derive(Debug, Queryable, Insertable, Selectable)]
+#[diesel(table_name = bsp)]
+pub struct Bsp {
```

Comment:
What do you think about storing the stake of the BSP here? I can't think of an immediate use case, but I think it would also be handy.

Reply:
Also, any reason why you're not storing the forest root of the BSP here?

Reply:
I will track once I process the events which emit (or should emit) the forest roots.